package.json

```diff
@@ -37,6 +37,11 @@
     "rollup": "^2.30.0"
   },
   "dependencies": {
-    "express": "^4.16.4"
+    "alert": "^5.0.5",
```

Comment:
Looks like this package is not used in the live build and so should not be listed here (this is more significant than it sounds)
```suggestion
```